RecoLocalMuon/MuonSysAging/plugins/GEMChamberMasker.cc

```diff
@@ -0,0 +1,172 @@
+// system include files
+#include <memory>
+#include <string>
+#include <vector>
+#include <iostream>
+#include <algorithm>
+#include <regex>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/GEMDigi/interface/GEMDigiCollection.h"
+#include "DataFormats/GEMDigi/interface/GEMPadDigiCollection.h"
+#include "DataFormats/GEMDigi/interface/GEMCoPadDigiCollection.h"
+#include "DataFormats/MuonDetId/interface/GEMDetId.h"
+#include "CondFormats/RecoMuonObjects/interface/MuonSystemAging.h"
+#include "CondFormats/DataRecord/interface/MuonSystemAgingRcd.h"
+
+//
+// class declaration
+//
+
+class GEMChamberMasker : public edm::stream::EDProducer<>
+{
+
+   public:
+      explicit GEMChamberMasker(const edm::ParameterSet&);
+      ~GEMChamberMasker();
+
+      static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
+   private:
+      virtual void produce(edm::Event&, const edm::EventSetup&) override;
+      
+      virtual void beginRun(edm::Run const&, edm::EventSetup const&) override;
+      virtual void endRun(edm::Run const&, edm::EventSetup const&) override;
+
+      // ----------member data ---------------------------
+  edm::InputTag digiTag_;
+  bool GE11Minus_;
+  bool GE11Plus_; 
+  bool GE21Minus_;
+  bool GE21Plus_; 
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.